R/data-combined.R

```diff
@@ -65,13 +65,8 @@
 DataCombined <- R6::R6Class(
   classname = "DataCombined",
   inherit = Printable,
-
```

Comment:
You don't have to remove everything :D 

# public fields and methods ------------------------------------

# active bindings ---------------------------------------------------

# private -----------------------------------

# private fields-----------------------------------

are definitely useful.

Reply:
Added them back in.